index.js

```diff
@@ -50,4 +50,12 @@ types.forEach(function(type) {
   normalize[type] = normalize.bind(null, type);
 });
 
+normalize.date = function(value) {
+  function dateOrTimestamp(value) {
+    return typeof value === 'number' || value instanceof Date;
```

Comment:
our current vinyl-fs code (https://github.com/gulpjs/vinyl-fs/blob/master/lib/filter-since.js#L6-L8) checks `instanceof Number` also.  Please add that and a test for the `Number` constructor.